CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.8.0 (2020-02-18)
```

Comment:
Should we update the date? Or is yesterday's date fine?

Reply:
Done.